lib/internal/streams/operators.js

```diff
@@ -281,7 +281,7 @@ function drop(number, options) {
 }
 
 
-function take(number, options) {
+function take(number, options = {}) {
```

Comment:
We don't need to create a throw away object here.
```suggestion
function take(number, options = undefined) {
```